src/systemd/isc-dhcp-relay6.service

```
@@ -5,7 +5,8 @@ Wants=network-online.target
 RequiresMountsFor=/run
 ConditionPathExists=/run/dhcp-relay/dhcrelay6.conf
 After=vyos-router.service
-
+StartLimitBurst=10
+StartLimitIntervalSec=60
```

```suggestion
StartLimitIntervalSec=0
```
Just setting `StartLimitIntervalSec=0` should disable rate limiting.